### PR TITLE
Update mongodb chart dependency

### PR DESCRIPTION
**Pull request by andresmgot**
Ref: #577

### CODE REVIEW

File: chart/kubeapps/Chart.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+version: 0.3.3
 appVersion: DEVEL
 description: Kubeapps is a dashboard for your Kubernetes cluster that makes it easy to deploy and manage applications in your cluster using Helm
 icon: https://raw.githubusercontent.com/kubeapps/kubeapps/master/docs/img/logo.png
 keywords:
-- helm
-- dashboard
-- service catalog
-- deployment
+  - helm
```
Comment (andresmgot) [inline chart/kubeapps/Chart.yaml]: this new indentation was automatically generated by the vscode editor

Comment (prydonius) [reply chart/kubeapps/Chart.yaml]: It doesn't do that for me, do you have some sort of YAML lint extension or something? The rest of the chart (e.g. values.yaml) doesn't indent lists twice so we should keep it consistent with that for now.

File: chart/kubeapps/Chart.yaml [hunk 2]
```diff
@@ -1,19 +1,19 @@
 apiVersion: v1
 name: kubeapps
-version: 0.3.2
+version: 0.3.3
```
Comment (prydonius) [inline chart/kubeapps/Chart.yaml]: should it be a minor version bump since MongoDB is a minor version bump?

File: chart/kubeapps/Chart.yaml (hunk 1 above)
Comment (andresmgot) [reply chart/kubeapps/Chart.yaml]: I don't, it's the normal `editor.formatOnSave` feature. I will disable it 😞

File: chart/kubeapps/Chart.yaml (hunk 2 above)
Comment (andresmgot) [reply chart/kubeapps/Chart.yaml]: I don't know the criteria in charts but for example with npm modules if a dependency gets updated it doesn't affect to the main version. Following the semver notation this is a "bug fix" so it should be a patch version bump.

**Pull request merged**